llvm/lib/Target/AMDGPU/SILowerI1Copies.cpp

```diff
@@ -512,10 +512,6 @@ bool PhiLoweringHelper::lowerPhis() {
              DT->getNode(RHS.Block)->getDFSNumIn();
     });
 
-#ifndef NDEBUG
-    PhiRegisters.insert(DstReg);
```

Comment:
There is still a count call, and the member exists. I'm not sure I follow what removing this fixes, but this should remove all the parts